src/go/agwd/config/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "config",
+    srcs = ["config.go"],
+    importpath = "github.com/magma/magma/agwd/config",
+    visibility = ["//visibility:public"],
```

Comment:
Wonder if gazelle is inferring the public-ness by virtue of go module cross-usage? Like, if the project references the go module across directories, it must be public, otherwise private?

Reply:
Yes, so the BUILD files were partially generated by gazelle, but they didn't work at all so I had to modify them quite a bit. But the visibility field is one that gazelle inferred and they seem to be correct. I think you are right that it looks at how the modules are used in the project.